tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -184,6 +198,18 @@ def test_update_partial(self, cognite_client, new_transformation):
             and updated_transformation.query == retrieved_transformation.query == "SELECT * from _cdf.assets"
         )
 
+    def test_update_nonce(self, cognite_client, new_transformation):
```

Comment:
could you also add a test to update the transformations using Transformation object (besides TransformationUpdate which is used in this test)?